### DISCUSSION TITLE
cloud_firestore : Unable to install BoringSSL-GRPC (0.0.32)

### DISCUSSION THREAD

**Discussion by ironFlash2407** [firebase/flutterfire#12896] [Q&A]
I am trying to build an app using Flutter but while trying to implement the Firestore. I added cloud_firestore to the pubspec.yaml file but am unable to run my flutter project the error comes up as follows.
![Uploading Screenshot 2024-06-04 at 6.12.29 PM.png…]()

 when i try running pod install in the ios folder the error states the following. 
![Uploading Screenshot 2024-06-04 at 6.15.26 PM.png…]()

Please help

**Comment by toddmueller** [comment]
I'm having the same issue. Were you able to resolve it?
